README.md

```diff
@@ -1,15 +1,19 @@
-# WUDU*
+# WUDU
+
+#### - Weighted Unweighted Directed Undirected (or like a Vodoo doll ;) )
+
+Application that lets you create, save, visualise, analyse and modify 4 different types of graphs
 
-Application that lets you create, save, visualise, analyse and modify 4 different types of graphs (see *) 
 
 ## Features
 
-- Support 4 types of graphs (see *)
-- Use all in all - 9 algorithms
-- Store graphs anywhere you want: SQLite, Neo4J, JSON
+### All in all:
+- Support 4 types of graphs
+- Use all in all - 9 algorithms (SCC, bridges, shortest paths, cycles, layout, communities, etc. see [wiki])
+- Store graphs anywhere you want: SQLite, Neo4J, JSON (WIP) (see [wiki])
```

Comment:
I think this paragraph's name is redundant, cause these are just features like stated above, and why repeat **all in all** twice?

Reply:
resolved